server/plugins/renderer/renderer.module.js

```diff
@@ -359,6 +359,7 @@ internals.getTemplatePath = function (path, data) {
  */
 internals.getPencilResponse = function (data, request, response, configuration) {
     data.context.theme_settings = configuration.settings;
+    data.contenxt.template_engine = configuration.settings.template_engine || "handlebars-v3";
```

Comment:
`contenxt` is a typo. Also, I don't think we're going to want to put this in the "settings" object. I think we should put it into a top-level key in https://github.com/bigcommerce/cornerstone/blob/master/config.json.